py-polars/requirements-dev.txt

```diff
@@ -21,6 +21,7 @@ connectorx
 cloudpickle
 fsspec
 s3fs[boto3]
+gevent
```

Comment:
We have a `dependencies` module for handling optional dependencies.

We should probably have an `async` option then in `pyproject.toml` then with `gevent` as well.

Reply:
I'll add `gevent`, but don's see any use in `async`, since it only uses standard library.

Reply:
Added gevent to dependencies.py, made better ImportError.